app/src/main/java/com/shevchenkovtwo/homework/MoviesAdapter.kt

```diff
@@ -46,4 +44,13 @@ class MoviesAdapter(private var movies: List<Movie>) :
     override fun getItemCount(): Int {
         return movies.size
     }
-}
\ No newline at end of file
+
+    companion object Constants {
+        var selectedMovie: Movie? = null
```

Comment:
That is not the best solution. The static makes a lot of coupling and caching issues.